src/lib/components/workflow/dropdown-filter/workflow-datetime-filter.svelte

```diff
@@ -0,0 +1,233 @@
+      {#if custom}
+        <div class="flex flex-col">
+          <p class="text-sm">To</p>
+          <div class="flex gap-2">
+            <DatePicker
+              on:datechange={onStartDateChange}
+              selected={startDate}
+            />
+            <TimePicker on:timechange={onStartDateTimeChange} />
+          </div>
+        </div>
+        <div class="flex flex-col">
+          <p class="text-sm">From</p>
+          <div class="flex gap-2">
+            <DatePicker on:datechange={onEndDateChange} selected={endDate} />
+            <TimePicker on:timechange={onEndDateTimeChange} />
+          </div>
+        </div>
+        <div class="flex gap-2">
+          <Button on:click={onApply}>Apply</Button>
+          <Button variant="secondary" on:click={() => (custom = false)}
+            >Cancel</Button
+          >
+        </div>
+      {:else}
+        <div>
+          <div class="flex w-full flex-wrap">
+            <div class="flex w-1/2 flex-col border-b border-gray-300">
+              <Option value={'All Time'}>All Time</Option>
+            </div>
+            <div class="flex w-1/2 flex-col border-b border-gray-300">
+              <Option value={'Custom'}>Custom</Option>
+            </div>
+            {#each columnOrderedDurations as duration}
+              <div class="flex w-1/2 flex-col justify-center">
+                <Option value={duration}>{duration}</Option>
+              </div>
+            {/each}
+            <div class="flex w-full flex-wrap">
+              <div class="flex w-1/2 flex-col border-t border-gray-300">
+                <div
+                  class="time-label"
+                  class:active={timeField === 'StartTime'}
+                  on:click={() => onTimeFieldChange('StartTime')}
+                >
+                  <div class="mr-2 w-6">
+                    {#if timeField === 'StartTime'}
+                      <Icon name="checkmark" />
+                    {/if}
+                  </div>
+                  Start Time
+                </div>
+              </div>
+              <div class="flex w-1/2 flex-col border-t border-gray-300">
+                <div
+                  class="time-label"
+                  class:active={timeField === 'CloseTime'}
+                  on:click={() => onTimeFieldChange('CloseTime')}
+                >
+                  <div class="mr-2 w-6">
+                    {#if timeField === 'CloseTime'}
+                      <Icon name="checkmark" />
+                    {/if}
+                  </div>
+                  End Time
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      {/if}
+    </div>
+  </Select>
+  <CustomSplitButton
+    class="rounded-tr rounded-br bg-offWhite"
+    buttonClass="border border-gray-900"
+    id="datetime"
+    label={capitalize($timeFormat)}
+    icon="clock"
+  >
+    <div on:click={() => ($timeFormat = 'relative')} class="timezone-label">
```

Comment:
Small UI thing, but what do you think about adding something here to indicate which one is selected?

![Screen Shot 2022-10-12 at 10 10 50 AM](https://user-images.githubusercontent.com/15069288/195400130-12025d9e-3661-4b9a-aafb-98f9a35d90b7.png)